dev-python/virtualenvwrapper/virtualenvwrapper-4.8.4_p20230121-r2.ebuild

```diff
@@ -51,7 +51,7 @@ src_prepare() {
 
 	# specify default python interpeter to align with PYTHON_SINGLE_TARGET
 	sed -i -e \
-		"s|\(_virtualenvwrapper_python_executable=\"\$(\).\w\((\"\)|\1command -v ${EPYTHON}\2|" \
+		"s|\(_virtualenvwrapper_python_executable=\"\$(command -v \).*\()\"\)|\1${EPYTHON}\2|" \
```

Comment:
Just use ${PYTHON} which is the absolute path.

Reply:
Done